src/utils/clmfates_interfaceMod.F90

```diff
@@ -1958,12 +2014,6 @@ subroutine wrap_accumulatefluxes(this, nc, fn, filterp)
                                this%fates(nc)%bc_out, &
                                dtime)
 
-    
-    call this%fates_hist%update_history_prod(nc, &
```

Comment:
Why was this removed instead of being changed to `update_history_hifrq`?  Is is redundant since this ends up being called later in another routine?

Reply:
hifrq now handles all high frequency FATES history writes, whereas before we split it into two different calls, which were doing very similar things.